src/controllers/osc/oscenumerator.cpp

```diff
@@ -0,0 +1,16 @@
+#include "controllers/osc/oscenumerator.h"
+
+OscEnumerator::OscEnumerator() {
+    m_devices.push_back(new OscController());
```

Comment:
This seems strange to me. I understand this to mean that when an OSCEnumerator is instantiated, it creates a single OSCController. The intent of the controller subsystem is to query available controllers in queryDevices() using whatever API is required to see what OSC endpoints are on the network and return that list there. That allows users to hit a 're-scan' button in the GUI which would call queryDevices() again to get an updated list. As well, there is supposed to be one OSCController instance per endpoint. Does OSC not fit this paradigm?